.github/workflows/iroha2-dev-pr.yml

```diff
@@ -137,3 +137,70 @@ jobs:
           file: Dockerfile
           # This context specification is required
           context: .
+
+  client-cli-test:
+    runs-on: ubuntu-latest
+    container:
+      image: hyperledger/iroha2-ci:nightly-2023-06-25
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Archlinux environment
+        run: |
+          sudo pacman -Syy
+          sudo pacman -S archlinux-keyring --noconfirm --disable-download-timeout
+          sudo rm -rf /etc/pacman.d/gnupg/* && sudo pacman-key --init && sudo pacman-key --populate archlinux
+          sudo pacman -Syu --noconfirm --disable-download-timeout
+
+      - name: Set up Rust toolchain and required dependencies
+        run: |
+          sudo pacman -S rustup mold musl rust-musl wget --noconfirm --disable-download-timeout
+          rustup toolchain install nightly-2023-06-25
+          rustup default nightly-2023-06-25
+          rustup target add x86_64-unknown-linux-musl wasm32-unknown-unknown
+          rustup component add rust-src
+          wget -c http://musl.cc/x86_64-linux-musl-native.tgz -O - | tar -xz
+          sudo ln -s /x86_64-linux-musl-native/bin/x86_64-linux-musl-g++ /x86_64-linux-musl-native/bin/musl-g++
```

Comment:
Those should already be installed in the container! Consider adding anything that you miss into the next container, so we don't waste valuable time in the CI each time we run the workflow.

Reply:
did it